### DISCUSSION TITLE
Not seeing Agentflows beta menu item after update

### DISCUSSION THREAD

**Discussion by rob-grissom** [FlowiseAI/Flowise#2715] [General]
I do not see the Agentflows beta menu item showcased in Leon van Zyl's most recent tutorial on building Multi-Agent teams.

Tried updating then reinstalling Flowise and updating again.

Flowise install is at 1.8.2 but I do not see the Agentflows Beta menu option.

Do I need to run a different command or is there a feature I need to update?

**Comment by amitach** [comment]
Its available in 1.8.3, you can pull and try
